stores/src/main/java/com/issoft/training/stores/comparaton/ComparatorByPrice.java

```diff
@@ -0,0 +1,13 @@
+package com.issoft.training.stores.comparaton;
+
+import com.issoft.training.domain.shop.Product;
+
+import java.util.Comparator;
+
+public class ComparatorByPrice implements Comparator<Product> {
```

Comment:
you already have a product comparator that can handle top sorting, think about using one class for that.